manifests/0000_25_kube-controller-manager-operator_06_deployment.yaml

```diff
@@ -44,11 +44,13 @@ spec:
           name: serving-cert
         env:
         - name: IMAGE
-          value: quay.io/openshift/origin-hyperkube:v4.0
+          value: quay.io/openshift/origin-hyperkube:latest
         - name: OPERATOR_IMAGE
-          value: docker.io/openshift/origin-cluster-kube-controller-manager-operator:v4.0
+          value: quay.io/openshift/origin-cluster-kube-controller-manager-operator:latest
         - name: CLUSTER_POLICY_CONTROLLER_IMAGE
-          value: quay.io/openshift/origin-cluster-policy-controller:v4.3
+          value: quay.io/openshift/origin-cluster-policy-controller:latest
+        - name: TOOLS_IMAGE
+          value: quay.io/openshift/origin-tools:latest
```

Comment:
This value is a regex placeholder for replacement mapped by https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/manifests/image-references.  We need to add the tools image to that `ImageStream` (and revert all the other changes to existing image values in this file, I assume were for testing)